sphinx/workshop/part2.py

```diff
@@ -23,7 +23,9 @@
 
 # Is it hot or is it cold?.
 
-if temp_avg > 27:
+if temp_avg > 30:
     print("It's pretty warm tonight")
+elif temp_avg > 26:
```

Comment:
`elif` is unnecessary, I think. It's just syntactic sugar over nested if/else. Basically, I want to cover only what is completely necessary.